jogoDoGalo.html

```diff
@@ -1,34 +1,35 @@
 <!DOCTYPE html>
 
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title> Jogo do Galo</title>
-    <link rel="stylesheet" href="ticTacToe.css" />
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title> Jogo do Galo</title>
+  <link rel="stylesheet" href="ticTacToe.css" />
+</head>
 
 <body>
 
-<div class="container">
-  
-<h1 class="titulo"> <h1> Jogo do Galo</h1> </a>
-
-<div id="vencedor" style="padding:20px"> Resultado </div>
-  <div class="area-jogo">
-    <div id="block_0" class="block" onClick="jogada(0)"></div>
-    <div id="block_1" class="block" onClick="jogada(1)"></div>
-    <div id="block_2" class="block" onClick="jogada(2)"></div>
-    <div id="block_3" class="block" onClick="jogada(3)"></div>
-    <div id="block_4" class="block" onClick="jogada(4)"></div>
-    <div id="block_5" class="block" onClick="jogada(5)"></div>
-    <div id="block_6" class="block" onClick="jogada(6)"></div>
-    <div id="block_7" class="block" onClick="jogada(7)"></div>
-    <div id="block_8" class="block" onClick="jogada(8)"></div>
+  <div class="container">
+
+    <h1 class="titulo">
+      <h1> Jogo do Galo</h1> </a>
```

Comment:
We need to change this line to stop using that `</a>` tag, can you do it?

Reply:
you're right, it doesn't make any sense. I'm going to delete it